src/tests/unit/tests/DetailsView/reports/components/report-sections/details-section.test.tsx

```diff
@@ -2,13 +2,26 @@
 // Licensed under the MIT License.
 import { shallow } from 'enzyme';
 import * as React from 'react';
-
+import { IMock, Mock, MockBehavior } from 'typemoq';
+import { DateProvider } from '../../../../../../../common/date-provider';
 import { DetailsSection, DetailsSectionProps } from '../../../../../../../DetailsView/reports/components/report-sections/details-section';
 
 describe('DetailsSection', () => {
     it('renders', () => {
+        const scanDate = new Date(Date.UTC(2018, 2, 9, 9, 48));
+
+        const getUTCStringFromDateMock: IMock<(date: Date) => string> = Mock.ofInstance(
+            DateProvider.getUTCStringFromDate,
+            MockBehavior.Strict,
+        );
+
+        getUTCStringFromDateMock
+            .setup(x => x(scanDate))
```

Comment:
`.setup(x => x(scanDate))`, can you rename `x` to something that better convey meaning? `getter` may work

Reply:
Sure. Should have done it when writing it. Thanks for the feedback. 👍